koku/api/query_params.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, request, caller, **kwargs):
 
         self.kwargs = kwargs
         self.request = request
+        self.caller = caller
```

Comment:
Yes! This is used in the query_handler on the parameters instance in order to configure rbac. When grouping by `org_unit_id` - we are actually combining 1. a query that groups by accounts and filters on `org_unit_id` and 2. A query filtering on each sub org with no group by - Since we are manipulating the query params outside of what the user has passed in, rbac is initially not configured so we have to call that [here](https://github.com/project-koku/koku/pull/2139/files#diff-9fa2e6770cbc217b865ad5a5f33ce5bcR214).

Reply:
Thanks for pointing that out!